gslib/utils/cat_helper.py

```diff
@@ -136,6 +136,13 @@ def CatUrlStrings(self,
               print('==> %s <==' % blr)
               printed_one = True
             cat_object = blr.root_object
+            # This if statement ensures nothing is ouputted and no error
+            # is thrown if the user enters an out of bounds range for the object.
+            if getattr(
+                cat_object, 'size', None
```

Comment:
What should the correct behavior here be if `cat_object.size` is `None`? In what scenario would that happen?

You might be able to replace this if statement with something like:
```
if getattr(cat_object, 'size', float('inf')) <= start_byte:
```

Reply:
done.